buildSrc/src/main/kotlin/Library.kt

```diff
@@ -13,17 +13,21 @@ object Library {
     const val room: String = "androidx.room:room-ktx:${Version.room}"
     const val androidxJUnit: String = "androidx.test.ext:junit:${Version.androidxJUnit}"
     const val espresso: String = "androidx.test.espresso:espresso-core:${Version.espresso}"
+    const val viewModel: String =
+        "androidx.lifecycle:lifecycle-viewmodel-ktx:${Version.lifecycle}"
 
     // Google
     const val material: String = "com.google.android.material:material:${Version.material}"
+    const val daggerHiltAndroid: String = "com.google.dagger:hilt-android:${Version.daggerHilt}"
+    const val hiltCompiler: String = "com.google.dagger:hilt-android-compiler:${Version.daggerHilt}"
```

Comment:
The current hilt compiler doesn't have this name anymore. It's good to pay attention to such things like the current names of some 3rd party deps in order to prevent some nerve racking bugs

Reply:
Okay thank you very much